README.md

```diff
@@ -20,16 +20,37 @@ $ html-sketchapp --file sketch.html --out-dir dist/sketch
 
 ## Install
 
-*Please note: html-sketchapp-cli targets the latest ["Active LTS" version](https://github.com/nodejs/Release#release-schedule) of Node. Older versions of Node are unsupported.*
+*Please note: html-sketchapp-cli requires [Node.js](https://nodejs.org/) and targets the latest ["Active LTS" version](https://github.com/nodejs/Release#release-schedule). Older versions of Node are unsupported.*
+
+To install html-sketchapp-cli globally, along with html-sketchapp's Sketch plugin:
 
 ```bash
 $ npm install --global html-sketchapp-cli
+$ html-sketchapp install
 ```
 
-Then, install the Sketch plugin.
+However, when using html-sketchapp-cli in the context of a project, you should install it locally instead:
```

Comment:
I think you could be a bit more explicit on when you'd install locally or globally, and maybe put the preferred one first?

Reply:
Updated the wording 👍